.changes/v3.0.0/714-features.md

```diff
@@ -0,0 +1,12 @@
+* Add trusted certificate management types `TrustedCertificate` and `types.TrustedCertificate`
+  together with `VCDClient.AutoTrustCertificate`, `VCDClient.CreateTrustedCertificate`,
+  `VCDClient.GetAllTrustedCertificates`, `GetTrustedCertificateByName`,
+  `VCDClient.GetTrustedCertificateById`, `TrustedCertificate.Update`, `TrustedCertificate.Delete`
+  [GH-714]
+* vCenter management types `VCenter` and `types.VSphereVirtualCenter` adds Create, Update and Delete
+ methods: `VCDClient.CreateVcenter`, `VCDClient.GetAllVCenters`, `VCDClient.GetVCenterByName`,
+ `VCDClient.GetVCenterById`, `VCenter.Update`, `VCenter.Delete`, `VCenter.Refresh` [GH-714]
+* Add NSX-T Manager management types `NsxtManagerOpenApi`, `types.NsxtManagerOpenApi` and methods
+  `VCDClient.CreateNsxtManagerOpenApi`, `VCDClient.GetAllNsxtManagersOpenApi`,
+  `VCDClient.GetNsxtManagerOpenApiById`, `VCDClient.GetNsxtManagerOpenApiByName`,
+  `TmNsxtManager.Update`, `TmNsxtManager.Delete` [GH-714]
```

Comment:
Just confirming, no TODOs to add to this PR?

Reply:
Added a test that would fail now https://github.com/vmware/go-vcloud-director/pull/714/commits/2f7d8585bbe1ffd2b1a6263b093fc50e73f4eae1